app/controllers/auth_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AuthController < ApplicationController
+  respond_to :json
+  before_action :ensure_json!
+
+  def auth
+    request.format = :json
+    email = request.headers['X-User-email']
```

Comment:
Is there a reason you didn't just use basic auth here?

Reply:
No. I could rework that if you feel it's better to use that standard.

Reply:
I'll change this to using basic auth.